analyses/snv-callers/scripts/01-calculate_vaf_tmb.R

```diff
@@ -0,0 +1,177 @@
+# Run variant caller evaluation for a given MAF file.
+#
+# C. Savonen for ALSF - CCDL
+#
+# 2019
+#
+# Option descriptions
+#
+# -m : Path to MAF file to be analyzed. Can be .gz compressed. Give path relative
+#      to top directory, 'OpenPBTA-analysis'.
+# -b : File path that specifies the caller specific WGS BED regions file that is
+#      saved as a TSV but has CHR, start, and stop as it's first three columns
+#      Note that column names are ignored. Give path relative to top directory, 
+#      'OpenPBTA-analysis'.
+# -l : Label to be used for folder and all output. eg. 'strelka2'. Optional.
+#      Default is 'maf'
+#
+# Command line example:
+#
+# Rscript 01-calculate_vaf_tmb.R \
+# -m data/pbta-snv-strelka2.vep.maf.gz \
+# -b data/WGS.hg38.strelka2.unpadded.bed \
+# -l strelka2
+
+# Establish base dir
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Run set up script
+source(file.path(root_dir, "analyses", "snv-callers", "scripts", "00-set_up.R"))
+
+# Load library:
+library(optparse)
+
+################################ Set up options ################################
+
+# Set up optparse options
+option_list <- list(
+  make_option(opt_str = c("-m", "--maf"), type = "character", default = "none",
+              help = "Relative file path (assuming from top directory of 
+              'OpenPBTA-analysis') to MAF file to be analyzed. Can be .gz compressed.",
+              metavar = "character"),
+  make_option(opt_str = c("-b", "--bed_wgs"), type = "character",
+              default = "none", help = "File path that specifies the caller specific
+                BED regions file. Assumes from top directory, 'OpenPBTA-analysis'",
+              metavar = "character"),
+  make_option(opt_str = c("-l", "--label"), type = "character",
```

Comment:
Do you trust people to set this correctly? I worry that it will be too easy for people to accidentally overwrite directories with the wrong data.

Reply:
I have an `--overwrite option` to avoid this problem on the 02-run_eval.R script, unsure why I didn't add the same option here. Will do.